editors/code/package-lock.json

```diff
@@ -1212,6 +1571,93 @@
                 "minimist": "^1.2.5"
             }
         },
+        "mocha": {
+            "version": "7.1.2",
+            "resolved": "https://registry.npmjs.org/mocha/-/mocha-7.1.2.tgz",
+            "integrity": "sha512-o96kdRKMKI3E8U0bjnfqW4QMk12MwZ4mhdBTf+B5a1q9+aq2HRnj+3ZdJu0B/ZhJeK78MgYuv6L8d/rA5AeBJA==",
+            "dev": true,
+            "requires": {
+                "ansi-colors": "3.2.3",
+                "browser-stdout": "1.3.1",
+                "chokidar": "3.3.0",
+                "debug": "3.2.6",
+                "diff": "3.5.0",
+                "escape-string-regexp": "1.0.5",
+                "find-up": "3.0.0",
+                "glob": "7.1.3",
+                "growl": "1.10.5",
+                "he": "1.2.0",
+                "js-yaml": "3.13.1",
+                "log-symbols": "3.0.0",
+                "minimatch": "3.0.4",
+                "mkdirp": "0.5.5",
+                "ms": "2.1.1",
+                "node-environment-flags": "1.0.6",
+                "object.assign": "4.1.0",
+                "strip-json-comments": "2.0.1",
+                "supports-color": "6.0.0",
+                "which": "1.3.1",
+                "wide-align": "1.1.3",
+                "yargs": "13.3.2",
+                "yargs-parser": "13.1.2",
+                "yargs-unparser": "1.6.0"
+            },
```

Comment:
:-( :-( :-( 

Only semi-joking, but I wonder if it would be possible to move development of extension to deno, which has build-in bundler, test framework and formatter....

Reply:
I guess at the moment it might be quite complicated if even possible. 

Are this mocha  deps stops you from accepting this PR?

Reply:
We will have to wait until vscode migrates to deno, which I estimate will happen in ... *never*

Reply:
>We will have to wait until vscode migrates to deno,

Not necessary -- vscode extension is a zip archive with some files, there's no node.js specific magic we need to produce one.

>Are this mocha deps stopping you from accepting this PR?

No, that's the same thing that https://github.com/Microsoft/vscode-languageserver-node/ is using, so I don't think its reasonable to avoid it. It's just my general dissatisfaction with the fact that I spend way more time on TypeScript build than on Rust build, despite the actual amount of TypeScript code being negligible :)